contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/plugins/image-upload/image-md-to-html.js

```diff
@@ -17,16 +17,28 @@ const IMAGE_REGEX = /!\[([^\]]*)]\(([^/]+\/([^\s]+))(?:\s=([0-9.]+)x([0-9.]+))*\
 export default markdown => {
```

Comment:
There are no changes in logic, I just reorganized it a bit to be able to read the regex easier